src/compo/viirs_l1bnc2ioda.py

```diff
@@ -146,6 +157,9 @@ def _read(self):
             solar_aa = geo_ncd.groups['geolocation_data'].variables['solar_azimuth'][:].data.ravel()
             sensor_za = geo_ncd.groups['geolocation_data'].variables['sensor_zenith'][:].data.ravel()
             sensor_aa = geo_ncd.groups['geolocation_data'].variables['sensor_azimuth'][:].data.ravel()
+            sin_va = np.sin(sensor_za*deg2rad) * earth_radius / (earth_radius + orbit_height)
+            sensor_va = np.arcsin(sin_va) * rad2deg
```

Comment:
ideally this should be a function as it is generic for all satellite observations that need to derive this

let me get back to you shortly on this @weiwilliam

Reply:
yes would be good to have something like a util.py in tool or something like that,